recipes/LibriSpeech/ASR/transformer/hparams/train_hf_whisper.yaml

```diff
@@ -87,7 +88,7 @@ augmentation: !new:speechbrain.lobes.augment.TimeDomainSpecAugment
 whisper: !new:speechbrain.lobes.models.huggingface_whisper.HuggingFaceWhisper
     source: !ref <whisper_hub>
     freeze: !ref <freeze_whisper>
-    save_path: !ref <save_folder>/whisper_checkpoint
+    save_path: !ref <whisper_folder>/whisper_checkpoint
```

Comment:
this will have a double whisper_checkpoint folder

Reply:
How do you mean?

Reply:
done